plans/upstream-keylime-containers.fmf

```diff
@@ -0,0 +1,33 @@
+summary:
+  Tests used by Packit/TFT CI on Github to test upstream keylime containers
+
+environment+:
+  TPM_BINARY_MEASUREMENTS: /var/tmp/binary_bios_measurements
+  KEYLIME_RUST_CODE_COVERAGE: 1
+  AGENT_DOCKERFILE: Dockerfile.upstream.c9s
```

Comment:
Why do we define dockerfile for agent and not for other services? Do we need dockerfile definitions at all, given we define images?